HW03/anastasia_evsihneyeva/anastasia_evsihneyeva_hw_03_t_02.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Homework3
+  DATE = /\[(.*)\]/
+  FROM = /(.*)(?=\ - -)/
+  TO = %r{/te(.*)(?=\ H)}
+  def task2(logs)
+    return [] if logs.nil?
+
+    variable = logs.to_s.split("\n")
+    info = []
+    variable.each do |log|
+      next unless log.include?('POST') && log =~ /(.*) - - \[(.*)\] "(.*)" (.*)/
+
+      str = "DATE: #{DATE.match(log)[1]} FROM: #{FROM.match(log)} TO: #{TO.match(log).to_s.upcase}"
+      info << str
+    end
+    info
```

Comment:
You can use `select` and `map` to make it more readable
```suggestion
    logs.split("\n")
        .select { |log| log.include?('POST') && log =~ /(.*) - - \[(.*)\] "(.*)" (.*)/ }
        .map { |log| "DATE: #{DATE.match(log)[1]} FROM: #{FROM.match(log)} TO: #{TO.match(log).to_s.upcase}" }
```